src/github.rs

```diff
@@ -201,7 +201,7 @@ impl RepositoryExt for Repository {
         push_options.remote_callbacks(get_remote_callbacks(&config::CONFIG.gitlab));
 
         let refspec = format!(
-            "refs/heads/pr-{}/{}/{}:refs/heads/pr-{}/{}/{}",
+            "+refs/heads/pr-{}/{}/{}:refs/heads/pr-{}/{}/{}",
```

Comment:
Does the `+` need to be in front of the target? ie:

```
"refs/heads/pr-{}/{}/{}:+refs/heads/pr-{}/{}/{}"
```

Reply:
I don't believe so. The [git book says](https://git-scm.com/book/en/v2/Git-Internals-The-Refspec):

> The format of the refspec is, first, an optional +, followed by <src>:<dst>, where <src> is the pattern for references on the remote side and <dst> is where those references will be tracked locally. The + tells Git to update the reference even if it isn’t a fast-forward.